src/pages/IDO/index.tsx

```diff
@@ -0,0 +1,336 @@
+import { Button } from 'rebass/styled-components'
+import { darken } from 'polished'
+import { CardSection, DataCard, CardNoise, CardBGImage } from '../../components/earn/styled'
+//import { useAllProposalData, ProposalData, useUserVotes, useUserDelegatee } from '../../state/governance/hooks'
+//import { useAllProposalData, ProposalData, useUserDelegatee } from '../../state/governance/hooks'
+//import DelegateModal from '../../components/vote/DelegateModal'
+//import { useTokenBalance } from '../../state/wallet/hooks'
+//import { useActiveWeb3React } from '../../hooks'
+//import { PNG, ZERO_ADDRESS } from '../../constants'
+//import { JSBI, TokenAmount, ChainId } from '@pangolindex/sdk'
+//import { JSBI, TokenAmount} from '@pangolindex/sdk'
+//import { shortenAddress, getEtherscanLink } from '../../utils'
+//import Loader from '../../components/Loader'   // Will use when we actually fetch IDOs
+//import FormattedCurrencyAmount from '../../components/FormattedCurrencyAmount'
+
+
+//import avalaunchIcon from '../../assets/images/avalaunch-icon.png'
+import yaygamesIcon from '../../assets/images/yaygames-icon.png'
+//import penguinFiIcon from '../../assets/images/penguinfi-icon.png'
+import boofinanceIcon from '../../assets/images/boofinance-icon.jpeg'
+//import boofinanceLogo from '../../assets/images/boofinance-logo.png'
+//import avalaunchLogo from '../../assets/images/avalaunch-logo1.png'
+//import penguinFiLogo from '../../assets/images/penguinfinance-logo.png'
+//import poolzLogo from '../../assets/images/poolz-logo.png'
+//import starterLogo from '../../assets/images/starter-logo.png'
```

Comment:
1. lets remove not necessary images from src/assets folder
2. lets give meaningful name to images instead of `avalaunch-logo1.png`

Reply:
1. Removed all other logos and left only the IDO icons
2. Removed all other references of icons or logos and only left the IDO icons. I think those are self explanatory.